Scenarios/CustomizingALGoForGitHub.md

```diff
@@ -0,0 +1,301 @@
+
+```json
+  "Environments": [
+    "<EnvironmentName>"
+  ],
+  "DeployTo<EnvironmentName>": {
+    "EnvironmentType": "<EnvironmentType>"
+  }
+```
+
+You also need to create an AuthContext secret (called `<EnvironmentName>_AuthContext`) and a powershell script (named `.github/DeployTo<EnvironmentType>.ps1`), which defines the deployment functionality. Example:
+
+```powershell
+Param([Hashtable] $parameters)
+
+$parameters | ConvertTo-Json -Depth 99 | Out-Host
+$tempPath = Join-Path ([System.IO.Path]::GetTempPath()) ([GUID]::NewGuid().ToString())
+New-Item -ItemType Directory -Path $tempPath | Out-Null
+Copy-AppFilesToFolder -appFiles $parameters.apps -folder $tempPath | Out-Null
+Get-ChildItem -Path $tempPath -Filter *.app | Out-Host
+$authContext = $parameters.authContext | ConvertFrom-Json
+Write-Host "Token Length: $($authContext.Token.Length)"
+```
+
+In this example the AuthContext secret is assumed to contain a Token property. Read [this](https://aka.ms/algosettings#customdeployment) for more information.
+
+### Adding custom workflows
+
+If you add new workflows to the `.github/workflows` folder, which is unknown to AL-Go for GitHub, AL-Go will leave them un-touched. These workflows needs to follow standard GitHub Actions schema (yaml) and can be triggered as any other workflows. Example:
+
+```yaml
+name: 'Create Build Tag'
+
+on:
+  workflow_run:
+    workflows: [' CI/CD','CI/CD']
+    types: [completed]
+    branches: [ 'main' ]
+
+run-name: "[${{ github.ref_name }}] Create build tag"
+
+permissions: read-all
+
+jobs:
+  CreateTag:
+    if: github.event.workflow_run.conclusion == 'success'
+    runs-on: windows-latest
+    steps:
+      - name: mystep
+        run: |
+          Write-Host "Create tag"
+```
+
+It is recommended to prefix your workflows with `my`, `our`, your name or your organization name in order to avoid that the workflow suddenly gets overridden by a new workflow in AL-Go for GitHub. The above workflow is a real example from [here](https://github.com/microsoft/BCApps/blob/main/.github/workflows/CreateBuildTag.yaml).
+
+> [!CAUTION]
+> This workflow gets triggered when the CI/CD workflow has completed. Note that the name of the CI/CD workflow currently is prefixed with a space, this space will very likely be removed in the future, which is why we specify both names in this example. Obviously this workflow would break if we decide to rename the CI/CD workflow to something different.
+
+### Adding custom scripts
+
+You can add custom powershell scripts under the .github folder for repository scoped scripts or in the .AL-Go folder for project scoped scripts. Specially named scripts in the .AL-Go folder can override standard functionality in AL-Go for GitHub workflows. A list of these script overrides can be found [here](https://aka.ms/algosettings#customdeployment). Scripts under the .github folder can be used in custom workflows instead of using inline scripts inside the workflow.
```

Comment:
I believe that this link should point to the "Run-AlPipeline script override" section, right?
```suggestion
You can add custom powershell scripts under the .github folder for repository scoped scripts or in the .AL-Go folder for project scoped scripts. Specially named scripts in the .AL-Go folder can override standard functionality in AL-Go for GitHub workflows. A list of these script overrides can be found [here](https://aka.ms/algosettings#scriptoverrides). Scripts under the .github folder can be used in custom workflows instead of using inline scripts inside the workflow.
```